### PR TITLE
Sign petition

### PR DESCRIPTION
**Pull request by ceciliarivero**
- I've also added Scrivener to validate the sign petition form.

### CODE REVIEW

File: routes/campaigners.rb [hunk 1]
```diff
@@ -71,7 +61,7 @@ class Campaigners < Cuba
         petition.save
 
         # TODO: This isn't saving the language for reasons unknown, need to fix.
-        res.write(ak_connector.create_petition_page(
+        res.write(AK_CONNECTOR.create_petition_page(
```
Comment (EricBoersma) [inline routes/campaigners.rb]: Good change!

File: lib/countries.rb [hunk 2]
```diff
@@ -0,0 +1 @@
...auru", "Nepal", "Netherlands", "Netherlands Antilles", "New Caledonia", "New Zealand", "Nicaragua", "Niger", "Nigeria", "North Korea", "Northern Mariana Islands", "Norway", "Oman", "Pakistan", "Palau", "Palestine", "Panama", "Papua New Guinea", "Paraguay", "Peru", "Puerto Rico", "Philippines", "Poland", "Portugal", "Qatar", "Reunion", "Romania", "Russia", "Rwanda", "Samoa", "San Marino", "Sao Tome and Principe", "Saudi Arabia", "Senegal", "Serbia", "Seychelles", "Sierra Leone", "Singapore", "Slovakia", "Slovenia", "Solomon Islands", "Somalia", "South Africa", "South Korea", "South Sudan", "Spain", "Sri Lanka", "St. Helena", "St. Kitts-Nevis", "St. Lucia", "St. Pierre and Miquelon", "St. Vincent and the Grenadines", "Sudan", "Suriname", "Swaziland", "Sweden", "Switzerland", "Syria", "Taiwan", "Tajikistan", "Tanzania", "Thailand", "Togo", "Tonga", "Trinidad and Tobago", "Tunisia", "Turkey", "Turkmenistan", "Turks and Caicos Islands", "Tuvalu", "Uganda", "Ukraine", "United Arab Emirates", "United Kingdom", "United States", "Uruguay", "Virgin Islands, U.S.", "Uzbekistan", "Vanuatu", "Vatican", "Venezuela", "Vietnam", "Wallis and Futuna", "Western Sahara", "Yemen", "Zambia", "Zimbabwe"]
```
Comment (EricBoersma) [inline lib/countries.rb]: Missing a new line at the end of the file.

**Comment by EricBoersma** [comment]
I'm a bit confused about Scrivener. Just ending it to the view without hooking anything up on the front end allows it to do validation?

File: lib/countries.rb (hunk 2 above)
Comment (ceciliarivero) [inline lib/countries.rb]: Thanks! Fixed. :-)

**Pull request merged**